composer.json

```diff
@@ -21,7 +21,8 @@
         }
     ],
     "require": {
-        "php": "^8.2.0"
+        "php": "^8.2.0",
+        "illuminate/http": "^11.5"
```

Comment:
We need to move our requires into `require-dev` based on how we're building the deliverable.